### PR TITLE
Use term wireless instead of network

### PR DESCRIPTION
**Pull request by vashworth**
Rename variables, update comments, etc from `network` to `wireless` to keep it more uniform.

Also, move non-overriden messages related to device selection into the file they're used.

Part 7 in breakdown of https://github.com/flutter/flutter/pull/121262.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/flutter_tools/lib/src/base/user_messages.dart [hunk 1]
```diff
@@ -263,24 +263,14 @@ class UserMessages {
...
-      "matching '$deviceId'.";
-  String get flutterNoDevicesFound => 'No devices found.';
-  String get flutterNoSupportedDevices => 'No supported devices connected.';
   String flutterMissPlatformProjects(List<String> unsupportedDevicesType) =>
       'If you would like your app to run on ${unsupportedDevicesType.join(' or ')}, consider running `flutter create .` to generate projects for these platforms.';
-  String get flutterFoundButUnsupportedDevices => 'The following devices were found, but are not supported by this project:';
-  String flutterFoundSpecifiedDevices(int count, String deviceId) =>
-      'Found $count devices with name or id matching $deviceId:';
-  String flutterChooseDevice(int option, String name, String deviceId) => '[$option]: $name ($deviceId)';
-  String get flutterChooseOne => 'Please choose one (or "q" to quit)';
```
Comment (vashworth) [inline packages/flutter_tools/lib/src/base/user_messages.dart:276]: Moved messages as per this comment from original PR: https://github.com/flutter/flutter/pull/121262/files#r1115130107

File: packages/flutter_tools/lib/src/mdns_discovery.dart [hunk 2]
```diff
@@ -57,7 +57,7 @@ class MDnsVmServiceDiscovery {
   /// The [deviceVmservicePort] parameter may be used to specify which port
   /// to find.
   ///
-  /// The [isNetworkDevice] parameter flags whether to get the device IP
+  /// The [useDeviceIPAsHost] parameter flags whether to get the device IP
```
Comment (jmagman) [inline packages/flutter_tools/lib/src/mdns_discovery.dart:60]: I read this as `use Device IPAs Host` (like an .ipa) which is confusing but I don't have a better suggestion.

**Review by jmagman — APPROVED** [review]

**Comment by christopherfujino** [comment]
@vashworth are you ready to merge this one?

**Comment by jmagman** [comment]
> @vashworth are you ready to merge this one?

She'll follow up next week.

**Pull request merged**